python/gstaichi/lang/kernel_impl.py

```diff
@@ -614,16 +624,19 @@ def __init__(self, _func: Callable, autodiff_mode: AutodiffMode, _classkernel=Fa
         impl.get_runtime().kernels.append(self)
         self.reset()
         self.kernel_cpp = None
-        self.compiled_kernels: dict[CompiledKernelKeyType, KernelCxx] = {}
+        self.materialized_kernels: dict[CompiledKernelKeyType, KernelCxx] = {}
```

Comment:
Could you clarify somewhere in the documentation what "materialization" means (as opposed to compiled)? I find this term very confusing.